packages/expo-sqlite/src/ExpoSQLiteNext.ts

```diff
@@ -1,43 +1,2 @@
-import type { SQLiteOpenOptions } from './NativeDatabase';
-import type { DatabaseChangeEvent } from './SQLiteDatabase';
-
-export default {
-  NativeDatabase(
-    databaseName: string,
-    options?: SQLiteOpenOptions,
-    serializedData?: Uint8Array
-  ): void {
-    throw new Error('Unimplemented');
-  },
-
-  NativeStatement(): void {
-    throw new Error('Unimplemented');
-  },
-
-  async deleteDatabaseAsync(databaseName: string): Promise<void> {
-    throw new Error('Unimplemented');
-  },
-
-  deleteDatabaseSync(databaseName: string): void {
-    throw new Error('Unimplemented');
-  },
-
-  importAssetDatabaseAsync(
-    databaseName: string,
-    assetDatabasePath: string,
-    forceOverwrite: boolean
-  ): Promise<void> {
-    throw new Error('Unimplemented');
-  },
-
-  //#region EventEmitter implementations
-
-  addListener(eventName: string, listener: (event: DatabaseChangeEvent) => void) {
-    throw new Error('Unimplemented');
-  },
-  removeListeners() {
-    throw new Error('Unimplemented');
-  },
-
-  //#endregion
-};
+import { requireNativeModule } from 'expo';
+export default requireNativeModule('ExpoSQLiteNext');
```

Comment:
Ensure this works well on standard web without DOM components. The errors should be reasonable.

Reply:
Also if there's no more platform-specific module, then you can just move this out of it's own module and delete the extra import.

Reply:
looking this more for a little bit, i feel like it doesn't have a better way to balance good error messages and maintenance cost. the current implementation of having stub method for each native method is not scaled: https://github.com/expo/expo/blob/021ffdb31d1840fad9f8f1e71be450cadfcbc644/packages/expo-sqlite/src/ExpoSQLiteNext.ts. it would be easy to miss some methods from native into this file.
 
the current error for web without dom components would be `Cannot find native module ExpoSQLiteNext`, which is not perfect but fine.
i would also keep the file because it is shared for other references, e.g. https://github.com/expo/expo/blob/021ffdb31d1840fad9f8f1e71be450cadfcbc644/packages/expo-sqlite/src/hooks.tsx#L4C8-L4C18

let's find other systematic ways to deal with it across all expo modules, if that's a problem we want to resolve.